helpers/command.py

```diff
@@ -382,9 +382,9 @@ def start(cls, frontend_only=False):
                                 config.get_letsencrypt_repo_path())
 
         if dict_['maintenance_enabled']:
-                CLI.colored_print('Maintenance mode is enabled. To resume '
-                                  'normal operation, use `--stop-maintenance`',
-                                  CLI.COLOR_INFO)
+            CLI.colored_print('Maintenance mode is enabled. To resume '
+                              'normal operation, use `--stop-maintenance`',
+                              CLI.COLOR_INFO)
```

Comment:
If we're going to start reformatting older code, should we start using the `Black` formatting style as we are in kpi?

```python
CLI.colored_print(
    'Maintenance mode is enabled. To resume '
    'normal operation, use `--stop-maintenance`',
    CLI.COLOR_INFO,
)
```

Reply:
I guess that's maybe something for a separate PR to reformat the whole of `kobo-install` into the `Black` style...?

Reply:
Agreed.

Reply:
Just created a new issue for this at #149